docs/doc.md

```diff
@@ -36,23 +38,25 @@ class ButterFingersPerturbation(SentenceOperation):
 
 **Description:** The `README.md` file should clearly explain what the transformation is attempting to generate as well as the importance of that transformation for the specified tasks.
 
-**Paraphrasers and Data Augmenters:** Besides perturbations, we strongly encourage transformation methods which act like paraphrasers and data augmenters. For non-deterministic approaches, we encourage you to specify metrics which could provide an estimate of the quality of generation. It is preferred to have a high precision transformation generator as compared to a low accuracy one. And hence it's okay to selectively generate.
+**Paraphrasers and Data Augmenters:** Besides perturbations, we strongly encourage transformation methods which act like paraphrasers and data augmenters. For non-deterministic approaches, we encourage you to specify metrics which could provide an estimate of the quality of generation. It is preferred to have a high precision transformation generator as compared to a low accuracy one. And hence it's okay if your transformation selectively generates. If your transformation loads a deep-learning model, especially a heavy one (like BERT or T5 or their cousins), set the `heavy` variable to `True`.
  
-**Test Cases:** At least 5 examples should be added in the file `test.json` as test cases for every added transformation. These examples serve as test cases as well as provide a glimpse of what the transformation takes as input and what it generates.
+**Test Cases:** At least 5 examples should be added in the file `test.json` as test cases for every added transformation. These examples serve as test cases as well as provide a glimpse to reviewers of what your transformation generates. The format of `test.json` can be borrowed from the sample transformations [here.](../interfaces)
 
 **Languages other than English:** We also strongly encourage multilingual perturbations. All applicable languages should be specified in the list of “locales”.
  
 ## Writing a good transformation
-Writing a transformation is a creative process. Transformations could be generated both using machine learning as well as rule based models. While there is no hard core rule, a transformation is worth if it can augment training data qualitatively or be able to generate perturbations which could reveal places where models' performance suffers. One of the quick ways to contribute would be to add a copy of any of the existing transformations for a low-resource language.
+Writing a transformation is a creative process. Transformations could use both machine learning as well as rule based models. While there is no hard core rule, a transformation is worth if it can augment training data qualitatively or be able to generate perturbations which could reveal places where models' performance suffers. One of the quick ways to contribute would be to add a copy of any of the existing transformations for a low-resource language.
```

Comment:
While there is no hard ~core~ and fast rule, a transformation is ~worth~ useful if it can augment training data qualitatively or be able to generate perturbations which could reveal places where models' performance suffers. One of the quick ways to contribute would be to ~add a copy of~ extend any of the existing transformations ~for~ to a low-resource language.